include/functions.h

```diff
@@ -794,7 +794,7 @@ f32* func_800B8248(PosRot* param_1, ActorPlayer* param_2);
 // void func_800B82EC(void);
 // void func_800B83BC(void);
 // void func_800B83F8(void);
-s32 func_800B84D0(Actor* actor, GlobalContext* globalCtx);
+s32 func_800B84D0(Actor* actor, GlobalContext* globalCtx); // Actor_IsTalking
```

Comment:
Why not name the function?

Reply:
Because `z_actor` is reserved by somebody else, and that person may not me renaming their reserved stuff.